cmd/kubebuilder/create/resource/run.go

```diff
@@ -61,6 +62,7 @@ func AddCreateResource(cmd *cobra.Command) {
 	createResourceCmd.Flags().BoolVar(&nonNamespacedKind, "non-namespaced", false, "if set, the API kind will be non namespaced")
 	createResourceCmd.Flags().BoolVar(&controller, "controller", true, "if true, generate the controller code for the resource")
 	createResourceCmd.Flags().BoolVar(&generate, "generate", true, "generate source code")
+	createResourceCmd.Flags().BoolVar(&coreControllerOnly, "core-controller-only", false, "skip resources, generate controller only")
```

Comment:
User specified at the project `init` time. Does he has to specify in each command now ? Looks like we should record this info in some .kb file.

Also, if does not specify `controller-only` during resource create in a `controller-only` project, what is the behavior ?